src/coprocessor/dag/executor/scanner.rs

```diff
@@ -81,13 +96,16 @@ impl<S: Snapshot> Scanner<S> {
     pub fn reset_range(&mut self, range: KeyRange, store: &SnapshotStore<S>) -> Result<()> {
         self.range = range;
         self.no_more = false;
+        unsafe {
+            self.seek_key.set_len(0);
```

Comment:
I think we should use a better name than `seek_key`, since we will not seek any more.